### PR TITLE
jenkins: Add Groovy script hash

### PR DESCRIPTION
**Pull request by kklin**
A previous commit updated the Groovy script to print the KELDA_VERSION
rather than QUILT_VERSION, but didn't update the script hash.

### CODE REVIEW

**Comment by aegamesi** [comment]
Where does this hash come from?

**Comment by kklin** [comment]
@aegamesi I updated the file with a comment saying how the hash was obtained.

File: config/jenkins/scriptApproval.xml [hunk 1]
```diff
@@ -1,7 +1,15 @@
 <?xml version='1.0' encoding='UTF-8'?>
+<!--
+This file defines the hashes of script that are allowed to execute. Jenkins
```
Comment (aegamesi) [inline config/jenkins/scriptApproval.xml]: 'of script => 'of scripts'

File: config/jenkins/scriptApproval.xml [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+requires an administrator to approve scripts before they are allowed to run.
+
+This file was generated by approving the script under Jenkins -> Manage Jenkins
+-> In-process Script Approval, and copying the resulting file at
```
Comment (aegamesi) [inline config/jenkins/scriptApproval.xml]: Might be better to make sure "Jenkins -> Manage Jenkins -> In-process Script Approval' is all on one line?

Comment (kklin) [reply config/jenkins/scriptApproval.xml]: Done

File: config/jenkins/scriptApproval.xml (hunk 1 above)
Comment (kklin) [reply config/jenkins/scriptApproval.xml]: And done 😄

**Pull request merged**